tests/contrib/test_multistore_file.py

```diff
@@ -186,9 +187,8 @@ def test_multistore_no_symbolic_link_files(self):
             'user-agent/1.0',
             ['some-scope', 'some-other-scope'])
         try:
-            self.assertRaises(
-                locked_file.CredentialsFileSymbolicLinkError,
-                store.get)
+            with self.assertRaises(CredentialsFileSymbolicLinkError):
```

Comment:
Oh, I see. Hmm... I think I'd prefer you to break assertRaises into multiple lines.